docs/how_tos/communicate-new-checks.rst

```diff
@@ -0,0 +1,53 @@
+How-To: Communicate New Lint Plugins or Checks
+==============================================
+
+Adding a new lint plugin
+------------------------
+
+After coding a new lint plugin, it should start off being optional, even if you intend to make it required-by-default in the future. See decisions/0001-adding-new-checks.rst for background on why.
+
+If all checks in the plugin will always be optional, simply provide instructions in the CHANGELOG.rst for how to add the plugin. For example::
+
+  * Explain the purpose of the new plugin, and potentially list its new checks.
```

Comment:
This is an instruction rather than an example.

Reply:
Thanks.  I'll fix.  I reformatted and combined sections, and  looks like I didn't fully fix this.

Reply:
This section has been removed/rewritten.